doc/feline.txt

```diff
@@ -858,6 +858,7 @@ Feline by default has some built-in providers to make your life easy. They are:
 │|feline-position|             │Get line and column number of cursor       │
 │line_percentage               │Current line percentage                    │
 │|feline-scroll_bar|           │Scroll bar that shows file progress        │
+│feline-search_count           │Current search total/current word count    │
```

Comment:
Do not modify Vim docs on your own, this will be autogenerated by a CI job. Modifying the USAGE.md file is enough